configs/detection/lfb/lfb_slowonly_nl.py

```diff
@@ -0,0 +1,150 @@
+_base_ = ['../_base_/models/slowonly_r50.py']
+
+# model settings
+lfb_prefix_path = 'data/ava/lfb_half'
+max_num_sampled_feat = 5
+window_size = 60
+lfb_channels = 2048
+dataset_modes = ['train', 'val']
+
+model = dict(
+    roi_head=dict(
+        shared_head=dict(
+            type='FBOHead',
+            lfb_cfg=dict(
+                lfb_prefix_path=lfb_prefix_path,
+                max_num_sampled_feat=max_num_sampled_feat,
+                window_size=window_size,
+                lfb_channels=lfb_channels,
+                dataset_modes=dataset_modes,
+                device='gpu'),
+            fbo_cfg=dict(
+                type='non_local',
+                st_feat_channels=2048,
+                lt_feat_channels=lfb_channels,
+                latent_channels=512,
+                num_st_feat=1,
+                num_lt_feat=window_size * max_num_sampled_feat,
+                num_non_local_layers=2,
+                st_feat_dropout_ratio=0.2,
+                lt_feat_dropout_ratio=0.2,
+                pre_activate=True)),
+        bbox_head=dict(in_channels=2560)))
+
+dataset_type = 'AVADataset'
+data_root = 'data/ava/rawframes'
+anno_root = 'data/ava/annotations'
+
+ann_file_train = f'{anno_root}/ava_train_v2.1.csv'
+ann_file_val = f'{anno_root}/ava_val_v2.1.csv'
+
+exclude_file_train = f'{anno_root}/ava_train_excluded_timestamps_v2.1.csv'
+exclude_file_val = f'{anno_root}/ava_val_excluded_timestamps_v2.1.csv'
+
+label_file = f'{anno_root}/ava_action_list_v2.1_for_activitynet_2018.pbtxt'
+
+proposal_file_train = (f'{anno_root}/ava_dense_proposals_train.FAIR.'
+                       'recall_93.9.pkl')
+proposal_file_val = f'{anno_root}/ava_dense_proposals_val.FAIR.recall_93.9.pkl'
+
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_bgr=False)
+
+mc_cfg = dict(
+    server_list_cfg='/mnt/lustre/share/memcached_client/server_list.conf',
+    client_cfg='/mnt/lustre/share/memcached_client/client.conf',
+    sys_path='/mnt/lustre/share/pymc/py3')
```

Comment:
: P

Reply:
:p, could I keep it until the task finished? Otherwise, it's more troublesome to change every time